linkcheck/htmlutil/formsearch.py

```diff
@@ -33,16 +33,8 @@ def add_value(self, key, value):
         self.data[key] = value
 
     def __repr__(self):
-        """Return unicode representation displaying URL and form data."""
-        return unicode(self)
-
-    def __unicode__(self):
-        """Return unicode string displaying URL and form data."""
-        return u"<url=%s data=%s>" % (self.url, self.data)
```

Comment:
seems like we lost the __str__ representation here.

Reply:
Yes, I was relying on:
`
If a class defines __repr__() but not __str__(), then __repr__() is also used when an “informal” string representation of instances of that class is required.
`

[https://docs.python.org/3/reference/datamodel.html#object.__repr__](https://docs.python.org/3/reference/datamodel.html#object.__repr__)

But I am not clear if there is test using this.

Reply:
oh wait, both representations are identical, nevermind. :)